homeassistant/components/usps.py

```diff
@@ -0,0 +1,124 @@
+"""
+Support for USPS packages and mail.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/usps/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_NAME, CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util.dt import now
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['myusps==1.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'usps'
+DATA_USPS = 'data_usps'
+SCAN_INTERVAL = timedelta(minutes=30)
+COOKIE = 'usps_cookies.pickle'
+
+CONF_INTERVAL = 'interval'
+DEFAULT_INTERVAL = 10
```

Comment:
Oh, gotcha. The name is a bit misleading given the precedent of the `interval` attribute.

You should not need to do `update()` in properties, or explicitly throttle anything, as long as `SCAN_INTERVAL` is present, iirc.